src/engine/profiler.cc

```diff
@@ -180,8 +184,15 @@ void Profiler::DumpProfile() {
 
 
 inline uint64_t NowInUsec() {
+#if defined(_MSC_VER) && _MSC_VER <= 1800
+  LARGE_INTEGER frequency, counter;
+  QueryPerformanceFrequency(&frequency);
+  QueryPerformanceCounter(&counter);
+  return counter.QuadPart * 1000000 / frequency.QuadPart;
```

Comment:
The Unit of frequency and counter is second, multiply 1000000 can get microsecond.